src/main/java/co/omise/models/Transfer.java

```diff
@@ -337,9 +337,36 @@ protected HttpUrl path() {
         }
     }
 
+    /**
+     * The {@link RequestBuilder} class for deleting a particular Transfer.
+     */
+    public static class DeleteRequestBuilder extends RequestBuilder<Transfer> {
+        private String transferId;
+
+        public DeleteRequestBuilder(String transferId) {
+            this.transferId = transferId;
+        }
+
+        @Override
+        protected HttpUrl path() {
+            return buildUrl(Endpoint.API, "transfers", transferId);
+        }
+
+        @Override
+        protected String method() {
+            return DELETE;
+        }
+
+        @Override
+        protected ResponseType<Transfer> type() {
+            return new ResponseType<>(Transfer.class);
+        }
+    }
+
     /**
      * The {@link RequestBuilder} class for destroying a particular transfer.
      */
+    @Deprecated
```

Comment:
Could you write a comment to refer to new function?

Reply:
Ohhh yes, how did I forget, will add it now.